src/feature_annotation/score_genes_cell_cycle_scanpy/test.py

```diff
@@ -0,0 +1,37 @@
+import mudata as mu
+
+input_file = f"{meta['resources_dir']}/merge_test_data/pbmc_1k_protein_v3_filtered_feature_bc_matrix_rna.h5mu"
+output_file = f"{meta['temp_dir']}/output.h5mu"
+
+# generate gene list files
+s_genes = ["MCM5", "PCNA", "TYMS"]
+s_genes_file = f"{meta['temp_dir']}/s_genes.txt"
+
+g2m_genes = ["UBE2C", "BIRC5", "TPX2"]
+g2m_genes_file = f"{meta['temp_dir']}/g2m_genes.txt"
+
+
+with open(s_genes_file, 'w') as file:
```

Comment:
Could you use pytest fixtures to create these files?
```python
@pytest.fixture
def s_genes_file(tmp_path):
    result = tmp_path / "s_genes.txt"
    s_genes = ["MCM5", "PCNA", "TYMS"]
    with result.open('w') as open_s_genes_file:
        open_s_genes_file.write("\n".join(s_genes))
    return result
```

You can use fixtures as arguments for tests:
```
def test_cell_scoring_cell_cycle(run_component, s_genes_file):
```